src/main/scala/cosas/ops/typeSets/Conversions.scala

```diff
@@ -107,88 +107,3 @@ object ToMap {
     def apply(s: D :~: T): Out =  rest(s.tail) + (key -> s.head.value)
```

Comment:
It's not exactly the same. It does not require any external implicits, for example.